airbyte-cdk/python/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 0.13.0
```

Comment:
Open Question: Do I need to describe this feature in the documentation anywhere? Tried to follow `ConnectionChecker` but don't see anything about it in any markdown files.

Reply:
Maybe add a section here? https://docs.airbyte.com/connector-development/cdk-python/http-streams

Probably worth documenting `HttpAvailabilityStrategy` but I'm not sure about the superclass.